src/main/resources/bootstrap.yaml

```diff
@@ -9,6 +9,11 @@ spring:
         ftp-private-key: FTP_PRIVATE_KEY
         ftp-public-key: FTP_PUBLIC_KEY
         upload-summary-report-recipients: UPLOAD_SUMMARY_REPORT_RECIPIENTS
+        send-letter-service-db-database-v11: DB_NAME
```

Comment:
so plan is later use this env vars in app yaml ?

Reply:
doesn't really matter. but for compatibility i guess we can keep previous one. resources will be overridden after the complete migration. migration for now will support 2 data sources. once migration is complete - we'll just flip config and be off with new db